docs/install-pmm/install-pmm-server/aws/run.md

```diff
@@ -1,59 +1,80 @@
-# Run the instance
+# Launch PMM Server on AWS
 
-After you add your new instance ,it will take some time to initialize it. When the AWS console reports that the instance is now in a running state, you may continue with configuration of PMM Server.
+After [installing PMM Server from AWS Marketplace](../aws/aws.md): 
+{.power-number}
 
-!!! note alert alert-primary "Note"
-    When started the next time after rebooting, your instance may acquire another IP address. You may choose to set up an elastic IP to avoid this problem.
+1. Wait until the AWS console reports that the instance is running. 
+2. Select your instance and open its IP address in a web browser. You can find the IP address in the **IPv4 Public IP** column or at the top of the **Properties** panel under the **Public IP** field:
+   
+    ![image](../../../_images/aws-marketplace.pmm.ec2.properties.png)
+3. Open the IP address in a web browser and log into PMM using the default credentials:
+     - Username: `admin`
+     - Password: `your instance ID`
+    ![image](../../../_images/PMM_Login.png)
+5. Change the default credentials then use the new ones on the PMM Server home page:
 
-With your instance selected, open its IP address in a web browser. The IP address appears in the **IPv4 Public IP** column or as value of the **Public IP** field at the top of the **Properties** panel.
+   ![PMM Home Dashboard](../../../_images/PMM_Home_Dashboard.png)
 
-![!image](../../../_images/aws-marketplace.pmm.ec2.properties.png)
+   These credentials not only manage access to the PMM web interface but also facilitate authentication between the PMM Server and PMM Clients. You will need to reuse these credentials when configuring PMM Clients on other hosts.
 
-To run the instance, copy and paste its public IP address into the location bar of your browser. In the **Percona Monitoring and Management** welcome page that opens, enter the instance ID.
+!!! note alert alert-primary ""
+    For SSH access instructions, see [Connecting to Your Linux Instance Using SSH](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/AccessingInstancesLinux.html). Replace the user name `ec2-user` with `admin`.
```

Comment:
Maybe we can mention here that SSH_key also can be added later on PMM Settings page too